config/xsl/common/sections/footer.xsl

```diff
@@ -2,9 +2,21 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
 
 <xsl:template name="footer">
-    <footer>
-        Copyright (c) 2005-2023 <a href="http://www.xceptance.com/" title="Visit the Xceptance website">Xceptance Software Technologies GmbH</a>
-    </footer>
+	<xsl:param name="productUrl" select="/testreport/configuration/version/productURL" />
+    <xsl:param name="productName" select="/testreport/configuration/version/productName" />
+    <xsl:param name="productVersion" select="/testreport/configuration/version/version" />
+    <div id="footer">
```

Comment:
Please keep using the `<footer>` tag instead of `div`.

Reply:
I did not even change this (someone else did that recently, I got it with rebase), but of course I can do that.

Reply:
Done.